src/App.tsx

```diff
@@ -1,61 +1,135 @@
 import './App.scss';
+import React, { FormEventHandler, useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const mapedTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const mapedUsers = usersFromServer.map(user => {
+  const { id, name } = user;
+
+  return (
+    <option value={id} key={id}>{name}</option>
+  );
+});
+
+export const App: React.FC = () => {
```

Comment:
Change `mapedTodos` and `mapedUsers` variable names to `mappedTodos` and `mappedUsers`.